npm-cli/src/Bindings.re

```diff
@@ -1,6 +1,21 @@
 [@bs.val] [@bs.module "fs"] external makedirSync: string => unit = "mkdirSync";
+[@bs.val] [@bs.module "fs"]
+external renameSync: (string, string) => unit = "renameSync";
+
+type stats;
+[@bs.val] [@bs.module "fs"] external statSync: string => stats = "statSync";
+
+[@bs.send] external isFile: stats => bool = "";
+
 [@bs.val] [@bs.module "process"] external cwd: unit => string = "cwd";
+[@bs.val] [@bs.module "process"] external argv: array(string) = "argv";
 [@bs.val] external scriptPath: string = "__dirname";
 [@bs.val] [@bs.module "child_process"]
 external exec: (string, (Js.Nullable.t(_), string, string) => unit) => unit =
   "";
+
+[@bs.module]
+external download_git: (string, string, option('a) => unit) => unit =
```

Comment:
Can we do something more light-weight? Downloading the git history seems unnecessary. We can work with this for the time being  though 👍

Reply:
It doesn't clone by default so it should just download the repo as a zip and then extract it. We could probably implement this ontop of the `download` package ourselves but this package allows us to easily download from gitlab and bitbucket as well.

Reply:
Oh, I misunderstood then. This is great!